service-clients-gen/src/main/java/com/flipkart/poseidon/serviceclients/idl/pojo/EndPointMetaInfoMeta.java

```diff
@@ -0,0 +1,31 @@
+package com.flipkart.poseidon.serviceclients.idl.pojo;
+
+import org.apache.commons.lang3.builder.EqualsBuilder;
+
+/**
+ * Created by jagannadha.raju on 14/09/18.
+ */
+public class EndPointMetaInfoMeta {
```

Comment:
not a good name, just metaInfo should be enough

Reply:
Just metaInfo is very generic. Incase, you want to have metadata for serviceIdl in future, then it might cause confusion then. I am going to make it EndpointMetaInfo. That should convey the required meaning